scripts/deblur

```diff
@@ -391,21 +422,23 @@ def build_biom_table(seqs_fp, output_biom_fp, min_reads, file_type, log_level, l
 @click.option('--overwrite', '-w', required=False, type=bool, default=False,
               is_flag=True,
               show_default=True, help="Overwrite output directory if exists.")
-@click.option('--read-error', '-e', required=False, type=float, default=0.05,
+@click.option('--read-error', required=False, type=float, default=None,
               show_default=True,
               help="Maximal bound on 1 nucleotide read error rate."
-                   "Required if --eror-dist is not supplied")
+                   "If supplied, overrides the --eror-dist!")
```

Comment:
`--error-dist`

Also, please omit the `!` as it is not polite to yell at the user